qt/applications/workbench/workbench/plugins/test/test_workspacewidget.py

```diff
@@ -66,6 +67,21 @@ def test_detector_table_shows_with_workspace(self):
             self.ws_widget._do_show_detectors([self.ws_names[0]])
         self.assert_widget_type_exists(MATRIXWORKSPACE_DISPLAY_TYPE)
 
+    @mock.patch('workbench.plugins.workspacewidget.plot', autospec=True)
+    def test_plot_with_plot_bin(self,mock_plot):
+        self.ws_widget._do_plot_bin([self.ws_names[0]], False, False)
+        mock_plot.assert_called_once_with(unittest.mock.ANY,errors=False, overplot=False, wksp_indices=[0],
```

Comment:
I guess you tried `assert_called_once_with(mtd[self.ws_names[0]],...` so do we have an issue comparing workspace instances here?

Reply:
Yes, that did not compare as the same, despite printing identical actual and expected responses.  As the ws was not too important here, I went for the ANY

Reply:
That's fine. I vaguely remember now when defining the wrappers that `==` could be ambigous as it could mean is this the same underlying object (do the pointers point to the same thing or do they have the same value (run CompareWorkspace). It requires some thought.

Reply:
This is fine for these purposes though.